models/message.js

```diff
@@ -10,17 +10,16 @@ const Message = DefineMap.extend("Message", {
 });
 
 Message.List = DefineList.extend("MessageList", {
-	"*": Message
+	"#": Message
 });
 
-Message.connection = superMap({
+Message.connection = realtimeRestModel({
```

Comment:
Any particular reason you're still using can-define here? Though it will work fine, it seems weird to use both can-observe and can-define.